assets/src/components/button/style.scss

```diff
@@ -0,0 +1,53 @@
+/**
+ * Button styles,
+ */
+
+.muriel-button {
+	text-align: center;
+	font-size: 14px;
+	line-height: 18px;
+	font-weight: 500;
+	color: $muriel-gray-700;
+	text-shadow: none;
+
+	&.is-button {
+		display: inline-block;
+		background-color: $muriel-white;
+		border-color: #CCCCCC;
+		border-radius: 3px;
+		cursor: pointer;
+		height: auto;
+		padding: 13px;
+		margin: 14px 7px;
+
+		&:active:enabled {
+			border-width: 2px 1px 1px;
+		}
+	}
+
+	&.is-primary {
+		border: none;
+		background-color: $muriel-hot-pink-500;
+		box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
+		color: $muriel-white;
+		text-shadow: none;
+
+		&:hover,
+		&:focus:enabled {
+			background-color: $muriel-hot-pink-400;
+			color: $muriel-gray-0;
+		}
+
+		&:active:enabled {
+			background-color: $muriel-hot-pink-700;
+			color: $muriel-white;
+		}
+	}
+
+	&.is-centered {
+		display: block;
+		width: 100%;
```

Comment:
In the mockups the centered Primary button doesn't fill full width. Adding `max-width` would achieve this. In Figma the button is 360px wide so that minus horizontal padding should do the trick:

<img width="524" alt="Screen Shot 2019-05-14 at 8 26 01 AM" src="https://user-images.githubusercontent.com/1477002/57698081-b1796780-7622-11e9-8ae9-186454346b61.png">

Reply:
Good spot, padding is set at 13px so in [cfaff9b06b64c3805e992f17728dba01b9b634a6] I've set a max width of 334px.